command/genesis/polybft_params.go

```diff
@@ -365,78 +370,78 @@ func (p *genesisParams) deployContracts(
 	polybftConfig *polyCommon.PolyBFTConfig,
 	chainConfig *chain.Chain,
 	burnContractAddr types.Address) (map[types.Address]*chain.GenesisAccount, error) {
-	type contractInfo struct {
-		artifact *artifact.Artifact
-		address  types.Address
+	var proxyAddresses []types.Address
+	for proxyAddr := range contracts.GetProxyImplementationMapping() {
+		proxyAddresses = append(proxyAddresses, proxyAddr)
 	}
```

Comment:
In order to fix the linting warning (define a slice capacity and use append).
WDYT?
```suggestion
	proxyToImplAddrMap := contracts.GetProxyImplementationMapping()
	proxyAddresses := make([]types.Address, 0, len(proxyToImplAddrMap))

	for proxyAddr := range proxyToImplAddrMap {
		proxyAddresses = append(proxyAddresses, proxyAddr)
	}
```

Reply:
fix 8dfce2f